src/scripts/python/TMB.py

```diff
@@ -0,0 +1,119 @@
+
+vcf = open(snakemake.input.vcf)
+artifacts = open(snakemake.input.artifacts)
+output_tmb = open(snakemake.putput.tmb, "w")
+
+
+FFPE_SNV_artifacts = {}
+header = True
+for line in artifacts:
+    if header:
+        header = False
+        continue
+    lline = line.strip().split("\t")
```

Comment:
maybe skip first line by reading it before the loop
```
next(artifacts)
for line in artifacts:
    lline = line.strip().split("\t")
```

Reply:
Fixed